### PR TITLE
Feature/last two sections

### PR DESCRIPTION
**Pull request by marcinel123**
Just ended last two sections and added proper styles. Please have a look. Thanks.

### CODE REVIEW

File: app/src/components/Footer/Footer.styles.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: flex-start;
+  }
+`;
+
+export const StyledLogo = styled.div`
+@media (min-width: 350px) {
+    margin-top: 40px;
```
Comment (AdamKniec) [inline app/src/components/Footer/Footer.styles.js:23]: indentation is wrong here

**Comment by AdamKniec** [comment]
One more thing, grey color in "Advanced statistics" section is incorrect. Please use exact color

File: app/src/components/Footer/Footer.styles.js (hunk 1 above)
Comment (marcinel123) [reply app/src/components/Footer/Footer.styles.js:23]: corrected with pritter

**Comment by marcinel123** [comment]
I did correct bright gray color and input form to be always in center position

**Pull request merged**